WordPress/Classes/Networking/Plugin Management/PluginManagementClient.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+protocol PluginManagementClient {
```

Comment:
This isn't in WPKit because I need the reference to `JetpackSiteRef`. If not for that this would live there.